index.html

```diff
@@ -39,6 +39,197 @@ <h1 class="greeting">I'm Ivan<br>Glad to meet <br>you!</h1>
+    <div class="text-box">
+      <div class="tonic-text">
+        <h1 class="tonic-title">Tonic</h1>
+
+        <ul class="info">
+          <li id="client">CANOPY</li>
+          <li id="role">Full stack Dev</li>
+          <li id="date"><span id="datetime3"></span></li>
+          <script>
+            var dt = new Date();
+            document.getElementById("datetime3").innerHTML = dt.getFullYear();
+          </script>
+        </ul>
+
+        <p>A daily selection of privately <br>personalized reads, no accounts or <br>sign-ups required.</p>
+
+        <div class="tags">
+          <ul class="tags-ul">
+            <li class="html-tag">html</li>
+            <li class="css-tag">css</li>
+            <li class="javaScript-tag">javaScript</li>
+          </ul>
+        </div>
+
+        <div class="project">
+          <button class="project-btn">View Project</button>
+        </div>
+
+      </div>
+
+      </div>
+    </div>
+
+    <div class="tonic-1">
+
+      <div class="multi2-img"></div>
+    <div class="text-box">
+      <div class="tonic-text">
+        <h1 class="tonic-title">Tonic</h1>
+
+        <ul class="info">
+          <li id="client">CANOPY</li>
+          <li id="role">Full stack Dev</li>
+          <li id="date"><span id="datetime4"></span></li>
+          <script>
+            var dt = new Date();
+            document.getElementById("datetime4").innerHTML = dt.getFullYear();
+          </script>
+        </ul>
+
+        <p>A daily selection of privately <br>personalized reads, no accounts or <br>sign-ups required.</p>
+
+        <div class="tags">
+          <ul class="tags-ul">
+            <li class="html-tag">html</li>
+            <li class="css-tag">css</li>
+            <li class="javaScript-tag">javaScript</li>
+          </ul>
+        </div>
+
+        <div class="project">
+          <button class="project-btn">View Project</button>
+        </div>
+
+      </div>
+
+      </div>
+    </div>
+
+  </section>
+
+  <section class="about-myself">
+
+    <div class="about-me">
+      <h1 class="about-title">About <br>Myself</h1>
+      <p class="about-descript">Hello, I'm a software developer! I can help you build a product, feature or website look at some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</p>
+      
+      <div class="social-media2">
+        <a class="socials" href="">LET'S CONNECT</a><br>
+
+        <ul class="social-links">
+          <li class="linkedin2"><a href="https://www.linkedin.com/in/ivan-ludick-11a055243/"><img src="images\Linkedin icon.jpg"></a></li>
+          <li class="gmail2"><a href="https://smiley20089@gmail.com"><img src="images\Gmail.jpg"></a></li>
+          <li class="github2"><a href="https://github.com/IvanLudick"><img src="images\GitHub.jpg"></a></li>
+        </ul>
+
+        <div class="resume">
+          <button class="resume-btn">Get my resume</button>
+        </div>
+      </div>
```

Comment:
- [ ] Kindly make sure to the same typography styles (text color, font weight, margin, padding, font family, etc, ....) for this section in order to meet the provided Figma design. :+1: 

## Figma design 

![image](https://user-images.githubusercontent.com/95297251/205093012-65cc6d9f-5e80-4033-bdcb-c220aeb3ac7e.png)

## your design 
![image](https://user-images.githubusercontent.com/95297251/205094022-f50f4ca8-5d7d-438a-b003-29d21b55cbe3.png)